source/assets/sass/components/_playground.scss

```
@@ -331,3 +331,30 @@
     text-align: right;
   }
 }
+
+// Edit on Playground Link
+.sl-c-to-playground {
+  border: 0;
+  display: var(--to-playground-display, none);
+  float: inline-end;
+  line-height: 1.2;
+  margin-block-start: var(--sl-gutter--quarter-negative);
+  padding: var(--sl-gutter--quarter);
+  text-align: end;
+  position: relative;
+  z-index: 1;
+}
+
+$syntaxes: ('scss', 'sass');
+
+@each $syntax in $syntaxes {
+  body[data-selected-syntax='#{$syntax}']
```

I don't think we need to include `body` here
```suggestion
  [data-selected-syntax='#{$syntax}']
```